lib/turn.rb

```diff
@@ -0,0 +1,23 @@
+class Turn
+
+  attr_reader :guess, :card
+  def initialize(guess, card)
+    @guess = guess
+    @card = card
+
+  end
+
+  def correct?
+    @guess == @card.answer
+  end
+
+  def feedback
+    if correct? == true
```

Comment:
since correct? will evaluate to true or false, you don't need the comparison operator here. You could have this line just read `if correct?`